src/main/scala/gestalt/api.scala

```diff
@@ -67,6 +68,11 @@ object api extends Toolbox {
     Ident.apply("java")
   }
 
+  def Ident(name: "_no_package_")(implicit dummy: Dummy2): TermTree = {
+    import options.unsafe
+    Ident.apply("<empty>")
+  }
+
```

Comment:
In both Dotty and Scala, there are two top packages `_root_` and `<empty>`. `<empty>` is introduced to avoid polluting the `_root_`.  To be more consistent with `_root_`, I think `_empty_` is better, and more friendly to use.